govc/library/info.go

```diff
@@ -222,7 +222,9 @@ func (r infoResultsWriter) writeItem(
 	fmt.Fprintf(w, "  Created:\t%s\n", v.CreationTime.Format(time.ANSIC))
 	fmt.Fprintf(w, "  Modified:\t%s\n", v.LastModifiedTime.Format(time.ANSIC))
 	fmt.Fprintf(w, "  Version:\t%s\n", v.Version)
-
+	if v.SecurityCompliance != nil {
```

Comment:
does similar check & print need to be added for the item security_compliance and certificate_verification_info?

Reply:
This is the the item `security_compliance`. Yes, I'll add `certificate_verification_info.status`, but leave out the cert chain (which is available via `-json`)